src/pages/Home/Home.page.jsx

```diff
@@ -1,38 +1,35 @@
-import React, { useRef } from 'react';
-import { Link, useHistory } from 'react-router-dom';
+import React, { useEffect, useState } from 'react';
 
-import { useAuth } from '../../providers/Auth';
-import './Home.styles.css';
+import Navbar from '../../components/Navbar';
+import SideMenu from '../../components/SideMenu/SideMenu.component';
+import CardVideoDisplayer from '../../components/CardVideo/CardVideo.component';
 
-function HomePage() {
-  const history = useHistory();
-  const sectionRef = useRef(null);
-  const { authenticated, logout } = useAuth();
+function HomePage({ theme, setTheme }) {
+  const [videos, setVideos] = useState([]);
 
-  function deAuthenticate(event) {
-    event.preventDefault();
-    logout();
-    history.push('/');
-  }
+  useEffect(() => {
+    const URL_MOCK_FILE =
```

Comment:
**Optional**:

Might make sense to move this constant to a `src/mocks.js` file